apps/app/src/app/pages/(components)/components/(sonner)/sonner.page.ts

```diff
@@ -46,6 +48,26 @@ export const routeMeta: RouteMeta = {
 				<spartan-code secondTab [code]="defaultCode" />
 			</spartan-tabs>
 
+			<spartan-section-sub-heading id="about">About</spartan-section-sub-heading>
+			<p class="${hlmP} mb-6">
+				Sonner is built on top of
+				<a
+					href="https://tutkli.github.io/ngx-sonner/"
+					target="_blank"
+					rel="noreferrer"
+					hlmBtn
+					variant="link"
+					class="${link}"
+				>
+					ngx-sonner
+				</a>
+				by
+				<a href="https://github.com/tutkli" target="_blank" rel="noreferrer" hlmBtn variant="link" class="${link}">
+					&#64;tutkli
+				</a>
+				.
+			</p>
+
```

Comment:
Added an About section to reference the third party library.

![CleanShot 2025-05-20 at 10 51 19](https://github.com/user-attachments/assets/d1bbc066-ebae-40c8-827d-16349088f284)